TTHAnalysis/macros/prepareEventVariablesFriendTree.py

```diff
@@ -263,12 +99,16 @@ def analyze(self,event):
     exit()
 
 if "{P}" in args[1]: args[1] = args[1].replace("{P}",args[0])
-if len(args) != 2 or not os.path.isdir(args[0]):
+if len(args) != 2:
     print "Usage: program <TREE_DIR> <OUT>"
     exit()
-if not os.path.isdir(args[1]): 
-    os.system("mkdir -p "+args[1])
-    if not os.path.isdir(args[1]): 
+if not os.path.isdir(args[0]):
+    print "Error. Input directory {input} does not exist".format(input=args[0])
+    exit()
+if not os.path.isdir(args[1]):
+    tempOut = args[1].replace('/pool/ciencias/','/pool/cienciasrw/')
```

Comment:
is this hack really needed? i.e. can't you run with the right path in `args[1]`? (also, the code will later write in `args[1]`, not `tempOut`... how can that work?)